godal.go

```diff
@@ -952,6 +1020,17 @@ func (ds *Dataset) ClearOverviews(opts ...ClearOverviewsOption) error {
 	return cgc.close()
 }
 
+// ClearStatistics delete dataset statisitics
```

Comment:
typos, needs list of available options, needs comment about minimal supported gdal version